### PR TITLE
Add @alexandruradovici to Core WG

### PR DESCRIPTION
**Pull request by alevy**
### Pull Request Overview

This pull request adds @alexandruradovici to the Core WG

### Testing Strategy

This was approved unanimously by the current Core WG

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### CODE REVIEW

**Review by bradjc — APPROVED** [review]

**Comment by bradjc** [comment]
Need link

File: doc/wg/core/README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ are to:
 - Amit Levy (chair), [alevy](https://github.com/alevy), Princeton University
 - Pat Pannuto, [ppannuto](https://github.com/ppannuto), UCSD
 - Leon Schuermann, [lschuermann](https://github.com/lschuermann), Princeton University
+- Alexandru Radovici [alexandruradovici], Politehnica Bucharest & OxidOS
```
Comment (brghena) [inline doc/wg/core/README.md:30]:
```suggestion
- Alexandru Radovici [alexandruradovici],(https://github.com/alexandruradovici) Politehnica Bucharest & OxidOS
```

**Review by brghena — APPROVED** [review]

**Comment by alevy** [comment]
Sorry @bradjc and @brghena, I accepted @brghena's addition of a link, so overrode your approvals

**Review by jrvanwhy — APPROVED** [review]

**Review by lschuermann — APPROVED** [review]

**Comment by alexandruradovici** [comment]
Thank you all for adding me to the core group 😁

**Pull request merged**